openstack/nova/templates/placement-ingress.yaml

```diff
@@ -7,8 +7,9 @@ metadata:
     system: openstack
     type: api
     component: nova
-  {{- if .Values.use_tls_acme }}
   annotations:
+    {{- include "utils.snippets.set_openstack_request_id" . | indent 4 }}
```

Comment:
Comparing this to the `api-ingress.yaml`, it looks like it's missing
> ingress.kubernetes.io/configuration-snippet: |

and needs more indent then, too?

Reply:
Indeed.